sky/compositor/color_filter_layer.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/compositor/color_filter_layer.h"
+
+namespace sky {
+namespace compositor {
```

Comment:
We usually leave a blank line under the last namespace declaration.  "git cl format" should clean up this sort of thing for you.